server/command.go

```diff
@@ -686,7 +686,7 @@ func executeInstanceList(p *Plugin, c *plugin.Context, header *model.CommandArgs
 
 	instances, err := p.instanceStore.LoadInstances()
 	if err != nil {
-		return p.responsef(header, "Failed to load known Jira instances: %v", err)
+		return p.responsef(header, "No Jira instances have been installed")
```

Comment:
I think the right place to handle this is in `LoadInstances()`. If we know that the `instances/v3` key is not in the store, `LoadInstances()` should return an empty `Instances` struct, with no error. Then the next line here `instances.IsEmpty()` will cover this case.

Looking through the code, I see that many calls to `LoadInstances()` does not have its error checked. Made a separate ticket https://github.com/mattermost/mattermost-plugin-jira/issues/722

Reply:
Sorry for being ambiguous with this request. The first paragraph in my comment above should be handled in this PR. The second paragraph does not need to be handled in this PR, and thus I have made a separate ticket just for that second paragraph. Does this make sense @Jwright707?

Reply:
I think I'm following. Would it be safe to say that this PR is ready to close? Sorry, this is my first open-source contribution and I'm still getting the hang of doing this. Thank you.